STM/jta/pom.xml

```diff
@@ -19,7 +19,9 @@
 	<version>5.10.5.Final-SNAPSHOT</version>
 	<packaging>jar</packaging>
         <properties>
-                <version.narayana>5.10.5.Final-SNAPSHOT</version.narayana>
+			<version.narayana>5.10.5.Final-SNAPSHOT</version.narayana>
```

Comment:
The indentation is somehow too far, isn't it?

+ please, consider this comment for other changes below as well

Reply:
okay